docs/.pages

```diff
@@ -5,4 +5,5 @@ nav:
   - how-to-guides
   - design
   - contributing
+  - datasets
```

Comment:
How about adding this under the `installation` chapter?
I guess people want to know about the datasets after installation.

If you want to add a new chapter, please also update the `introduction` page.

Reply:
I thought about putting it to `installation` and `tutorials` and even `how to guides` part too. But if someone (an outsider) just wants to find datasets, it is hard for them to figure out where it would be.

I also think we can enrich the page with outsider online open datasets in the future, it can be its own section.

I know that this documentation is only for "autoware" but I think it's good to have this kind of information available in a single place.

Reply:
> If you want to add a new chapter, please also update the introduction page.

On my way!

Reply:
Done in https://github.com/autowarefoundation/autoware-documentation/pull/200/commits/f96584899fcdb48970088e2fd7992ea81db8a8a3